src/runtime/virtcontainers/sandbox.go

```diff
@@ -248,6 +248,8 @@ type Sandbox struct {
 	seccompSupported  bool
 	disableVMShutdown bool
 	isVCPUsPinningOn  bool
+
+	hotplugNetworkConfigApplied bool
```

Comment:
it's a flag default to false, once hot-plugged network interface(VFIO in this case)'s network config applied successfully, it will be set to true. It's for preventing network config API being called multiple times when Pod has multiple containers.

PTAL comment at https://github.com/kata-containers/kata-containers/pull/9977/files#diff-6fb7f551035031ed29ad100c61f1a7e92b0fa15d2f67e4c7ea7ebf3b715583f4R1249-R1261

Reply:
@Apokleos thanks for reviewing this PR

Reply:
Could you please add comment to explain this introduced flag ?

Reply:
This PR almost looks good to me but the two comments !

Reply:
added docstring for the flag, PTAL.